client/app/components/ParameterValueInput.less

```diff
@@ -5,9 +5,15 @@
 .parameter-input {
   display: inline-block;
   position: relative;
+  width: 100%;
 
-  .@{ant-prefix}-input[type="text"] {
-    width: 195px;
+  .@{ant-prefix}-input,
+  .@{ant-prefix}-input-number {
+    min-width: 100% !important;
+  }
+
+  .@{ant-prefix}-select {
+    width: 100%;
```

Comment:
Fix this :beetle: (Select needs a `min-width`):
![min-width-search](https://user-images.githubusercontent.com/3356951/62495660-f689e180-b7ac-11e9-9c1a-a4885e0a198c.gif)

Reply:
Select doesn't have minWidth 60px?

Reply:
Yes, previously we were replacing that with the `!important` flag, so now it's flexible and has minWidth 60px